packages/netlify-cms-core/src/actions/__tests__/mediaLibrary.spec.js

```diff
@@ -171,7 +224,6 @@ describe('mediaLibrary', () => {
         });
 
         expect(backend.persistMedia).toHaveBeenCalledTimes(1);
-        expect(backend.persistMedia).toHaveBeenCalledWith(store.getState().config, assetProxy);

```

Comment:
What do you think on using:

```js
expect(backend.persistMedia).toHaveBeenCalledWith(
          store.getState().config,
          expect.objectContaining({
            path: 'static/media/name.png',
          }),
        );
```

Reply:
Good idea. I've added that assertion for the sanitization test as well.